packages/core/src/BinaryDataManager/FileSystem.ts

```diff
@@ -130,6 +77,7 @@ export class BinaryDataFileSystem implements IBinaryDataManager {
 	}
 
 	async deleteBinaryDataByExecutionIds(executionIds: string[]): Promise<string[]> {
+		// TODO: switch over to new folder structure, and delete folders instead
```

Comment:
Ideally similar to what we decided for S3: `/workflows/{workflowId}/executions/{executionId}/binary_data/{binaryDataFileNanoId}`

Reply:
I did not change this yet, because I'd like to try to make this backward-compatible, so 
1. that user instances don't end up with lots of old binary data files that never get cleaned up.
2. viewing binary data from older executions still works